### DISCUSSION TITLE
Ignore permitted applications if not present on the computer?

### DISCUSSION THREAD

**Discussion by JohanBTJ** [SafeExamBrowser/seb-win-refactoring#570] [General]
Goodday! 

Is there a way to ignore allowed applications when starting SEB if they are not present on the computer? For example, dyslexics may use a program that is on their computer, which is obviously not on the computer of others. The others are now getting error messages that the program is not present because of the config file. That would mean you have to create and release 2 config files? Are there any solutions here that I am not aware of?

Thanks in advance for thinking along!

**Comment by dbuechel** [comment]
At the moment, there is no such option, but apart from the error message, SEB should start normally on the systems which do not have the application(s) installed.

**Reply by JohanBTJ** [reply]
Yes, that's right: Seb starts normally on the computer.
The error only makes users and teachers a little nervous and then they wonder if everything is set up and working properly :)

Maybe a future feature 👍 

Thanks for the comment!

**Reply by dbuechel** [reply]
That is indeed a good point, I'll create a new feature request but most unfortunately, this will have to wait for one of the next planned versions as the backlog for the upcoming version 3.5.0 is already full.

**Reply by dbuechel** [reply]
See #572.

**Reply by JohanBTJ** [reply]
Awesome! Thank you @dbuechel !
